js/load-scene.js

```diff
@@ -61,23 +61,56 @@ AFRAME.registerComponent('loadscene', {
       lightPop.setAttribute('position', { x: -3.026, y: 4.122, z: -0.366 });
     }
 
+    var containerEl = document.getElementById('sts-live');
+    var sceneEl = document.querySelector('a-scene');
+    var introAudioEl = document.getElementById('intro-audio');
+    var ambienceAudio = document.getElementById('ambience');
+    var getReady = document.getElementById('get-ready');
+    var getStartedButton = document.getElementById('get-started');
 
-    document.querySelector('a-scene').addEventListener('loaded', function () {
-      document.getElementById('get-ready').style.display = 'none';
-      document.getElementById('intro-audio').play();
-      document.getElementById('ambience').play();
-      document.getElementById('camera-audio').addEventListener('loaded', function() { console.log('ok') })
+    var enterVREl;
+    var audioStarted = false;
+    var userPressEvent = 'ontouchstart' in window ? 'touchend' : 'mousedown';
+    var userPressDown = onUserPressDown.bind(this);
```

Comment:
is the `.bind(this)` necessary in this case? I think you can do `onUserPressDown()` and `onSceneLoaded()`